scripts/train.py

```diff
@@ -211,8 +218,10 @@ def dummy_init_fn(module: torch.nn.Module) -> None:
             param_init_fn=param_init_fn,
             **hybrid_sharding_fsdp_kwargs,
         )
-    elif cfg.distributed_strategy is None:
-        raise NotImplementedError("Single accelerator training not implemented yet!")
+    elif cfg.distributed_strategy == DistributedStrategy.single:
+        param_init_fn = None
+        olmo_model = olmo_model.to(device)
+        dist_model = SingleAccelerator(olmo_model)
```

Comment:
We still need to check for `None` somehow. If it's `None`, we throw a config exception.